pep-0001.txt

```diff
@@ -348,6 +393,11 @@ the interpreter unduly.  Finally, a proposed enhancement must be
 the Steering Council.  This logic is intentionally circular.)  See :pep:`2`
 for standard library module acceptance criteria.
 
+Except where otherwise approved by the Steering Council, pronouncements
+of PEP resolution will be posted to the `Python-Dev`_ mailing list,
+and may be mirrored on the official "Discussions-To" thread and other venues,
+as appropriate.
```

Comment:
```suggestion
Pronouncements
of PEP resolution will be posted to the `Python-Dev`_ mailing list.
```
I believe this is current practice, though the SC can change it if they want. I don't think we need to explicitly mention that the pronouncement can be mirrored elsewhere.

Reply:
> I believe this is current practice, though the SC can change it if they want.

It is except for packaging PEPs, which _de jure_ per the [SC standing delegations](https://github.com/python/steering-council/blob/main/process/standing-delegations.md), the PyPA governance PEP (PEP-0609) and the normative [PyPA governing documents](https://www.pypa.io/en/latest/specifications/), as well as _de facto_ on the recent packaging-related PEPs I checked, are always officially announced, discussed and pronounced on the Packaging category on Discourse, with the Distutils-SIG mailing list finally being fully shut down and formally retired.

> I don't think we need to explicitly mention that the pronouncement can be mirrored elsewhere.

Sure, fair enough. I eliminated those two lines.